js/query-parser.js

```diff
@@ -38,6 +38,21 @@ Fliplet.Registry.set('dynamicListQueryParser', function() {
   });
   this.queryPreFilter = _(this.pvPreFilterQuery).size() > 0;
 
+  // We can sort only by one column that is why this syntax dosn't supports
+  // ?dynamicListFilterColumn=Name,Age&dynamicListFilterOrder=asc
+  // Correct example is
+  // ?dynamicListFilterColumn=Name&dynamicListFilterOrder=asc
+  this.pvPreSortQuery = _.pickBy({
```

Comment:
can you explain what these `pvPreSortQuery` and `queryPreSort` do?

Reply:
@squallstar it was made like a filter query above and actually, all that these two variables do is:
1. `pvSortQuery` - is gathering query parameters from the `Fliplet. Navigate.query` method.
2. `queryPreSort` - determinates if the query parameters that we are looking at in the url are present.

Reply:
```suggestion
  this.pvSortQuery = _.pickBy({
```

I would ignore the "pre" part. Put this further down the code close to where the filter and search queries are.

"Prefilters" and "presorts" would have referred to filtering and sorting the data before rendering the LFD. That's not what the current